sdk/core/Azure.Core/src/SerializableOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure
+{
+    /// <summary>
+    /// Provides the client options for serializing models.
+    /// </summary>
+    public class SerializableOptions
+    {
+        /// <summary>
+        /// Bool that determines if ReadOnlyProperties will be serialized. Default is false.
+        /// </summary>
+        public bool IgnoreReadOnlyProperties { get; set; }
```

Comment:
Should this be `IncludeReadOnlyProperties` as used in the test and described in the summary?